scripts/integration-test.sh

```diff
@@ -131,7 +131,7 @@ function cleanup() {
   echo "INFO: Terminating background jobs"
   echo "  any kill usage errors below may be ignored"
 
-  for p in collector processor middleware
+  for p in collector processor
```

Comment:
The integration tests here only tested crash_data. If I remove crash_data from the middleware, there's no point keeping this in the tests because it never uses the middleware, because it can't work.